include/deal.II/base/mpi.h

```diff
@@ -500,6 +501,51 @@ namespace Utilities
                        const ArrayView<const T> &values,
                        const MPI_Comm           &mpi_communicator,
                        const ArrayView<T>       &output);
+      /**
+       * This function is intented for internal use in the library and
```

Comment:
it's accessible to users, so please remove "internal use" and just focus on what the function does.